src/main/resources/db/migration/V24__avstemmingsjobb.sql

```diff
@@ -0,0 +1,12 @@
+DO
+$$
+    BEGIN
+        IF NOT EXISTS
+            (SELECT 1 FROM task WHERE type = 'task.avstemming')
+        THEN
+            INSERT INTO task(payload, type, status, metadata, versjon, opprettet_tid, trigger_tid)
+            VALUES ('2020-09-20', 'task.avstemming', 'UBEHANDLET', 'callId=e5ac23de-21b7-497a-9be3-dbe1d8110088
```

Comment:
bør ikke dato være samme dag som du merger denne?

Reply:
Det viktigste er er at datoen er før vi merger. Hvis det ikke er noen forekomster blir det regnet som en vellykket kjøring og ny task opprettes for neste dag. Slik vil den spinne seg frem til dagens dato.